appveyor.yml

```diff
@@ -0,0 +1,60 @@
+version: '{build}-{branch}'
+platform:
+  - x64
+build: off
+max_jobs: 2
+branches:
+  only:
+    - master
+    - /^v\d/
+cache:
+- C:\downloads
+- C:\vendor\bundle
+init:
+- ps: $env:GEM_SOURCE = "http://rubygems.org"
+install:
+- set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+- ps: |
+    if (!(Test-Path C:\downloads)) { mkdir C:\downloads | Out-Null }
+    $log = "C:/puppet-agent.log"
+    $agent_url = "https://downloads.puppetlabs.com/windows/${ENV:PUPPET_REPO}/puppet-agent-${ENV:PUPPET_VERSION}-x64.msi"
+    Write-Output "Installing Puppet from $agent_url"
+    Write-Output "Log will be written to $log"
+    if ( Test-Path $log ) { Remove-Item $log }
+    cd C:\downloads
+    if (!(Test-Path "puppet-agent-${ENV:PUPPET_VERSION}-x64.msi")) { Start-FileDownload $agent_url }
+    Start-Process msiexec.exe -Wait -NoNewWindow -ArgumentList @("/i", "C:\downloads\puppet-agent-${ENV:PUPPET_VERSION}-x64.msi", "/qn", "/l*", "$log")
+    cd $ENV:APPVEYOR_BUILD_FOLDER
+- ps: Copy-Item -Path $ENV:APPVEYOR_BUILD_FOLDER -Destination C:/ProgramData/PuppetLabs/code/environments/production/modules/sensu -Recurse
+- ps: Copy-Item -Path "${ENV:APPVEYOR_BUILD_FOLDER}/tests/ssl" -Destination C:/ProgramData/PuppetLabs/puppet/etc/ssl -Recurse
+- ps: Copy-Item -Path "${ENV:APPVEYOR_BUILD_FOLDER}/lib/facter" -Destination C:/ProgramData/PuppetLabs/puppet/cache/lib/facter -Recurse
+- set PATH=C:\Program Files\Puppet Labs\Puppet\bin;%PATH%
+- puppet --version
+- puppet module install puppetlabs-stdlib
+- puppet module install puppet-archive
+- puppet config set --section main certname sensu_agent
+- facter -p --debug
+- ruby -v
+- gem update --system 2.7.9
+- gem -v
+- bundle -v
+- bundle install --jobs 4 --retry 2 --path C:\vendor\bundle
+test_script:
+- bundle exec rake acceptance:windows
+image:
```

Comment:
should also include windows 10, since that's what the developer used to write the windows support

Reply:
There is no Windows 10 with appveyor, only 2012R2 and 2016.  https://www.appveyor.com/docs/windows-images-software/#operating-system